qiskit/optimization/algorithms/admm_optimizer.py

```diff
@@ -547,43 +586,43 @@ def _create_step3_problem(self):
 
     # when a plain list() call is used a numpy type of values makes cplex to fail when cplex.write() is called.
     # for debug only, list() should be used instead
-    def _to_list(self, values):
+    @staticmethod
+    def _to_list(values):
```

Comment:
Rejected. Yes, this method is really required:
1. Current optimization API does not work with NumPy objects.
2. If you want to save a problem with NumPy objects, you get an exception from CPLEX.
3. _to_list is a very simple name and the implementation is very simple also. This method is used in the complex parts of the code, so I really don't want to make it inline.

Reply:
Do I understand correctly that you're using this method only to cast NumPy types to a list? If that's the case we can just use `tolist()` [1] instead of the method. What do you think?

[1]: https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.tolist.html

Reply:
Completely missed that this method casts numpy floats to python floats. That was the point initially, seems this is a convenient substitution. Will run more tests. Thanks a lot!